buildtools/src/main/resources/pmd/pmd.xml

```diff
@@ -9,7 +9,10 @@
         Trellis PMD rules
     </description>
 
-    <rule ref="category/java/bestpractices.xml"/>
+    <rule ref="category/java/bestpractices.xml">
+      <exclude name="JUnitTestContainsTooManyAsserts"/>
+      <exclude name="JUnitAssertionsShouldIncludeMessage"/>
```

Comment:
Is there a PMD switch for warning only? It might be good for this one. Most of the assertions are fairly self-explanatory, but messages _never_ hurt.

Reply:
The reason I turned this one off is because the warnings were for instances where there _were_ messages. I think the PMD code (and we're using the latest release) is looking for JUnit4 constructions where messages appear in the first position, unlike in JUnit5 constructions where messages appear at the end.

Reply:
Ahhhh, cool. Didn't know that.